api/get_balance.py

```diff
@@ -15,12 +15,12 @@ def get_msc_balances( addr ):
   #TODO move functionality for individual currencies into /tx/ endpoint (sent, received, total reserved balances, etc.)
   addr = re.sub(r'\W+', '', addr) #check alphanumeric
   ROWS=dbSelect("select * from addressbalances ab, smartproperties sp where ab.address=%s and ab.propertyid=sp.propertyid "
-                "and sp.protocol='Mastercoin'", [addr])
+                "and sp.protocol='Omni'", [addr])
```

Comment:
@Nevtep same here,   lets change 
and sp.protocol='Mastercoin'   ->   and (sp.protocol='Mastercoin' or sp.protocol='Omni')